### PR TITLE
[MM-8]: Added topic for starting meeting

### LINKED ISSUE #8: Add subject to meetings
Meetings can have a subject. How do we want to provide the subject?

Shall we use the same behaviour as Jitsi?

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
### Summary
Updated slash command to start a meeting to support adding topic to a meeting
### Screenshot
image

#### Ticket Link
Fixes #8

### CODE REVIEW

File: server/command.go
Comment (raghavaggarwal2308) [inline server/command.go:1]: We will have to change the meeting topic in the api call we are making for creating the meeting. In `meeting.go` file

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

**Review by raghavaggarwal2308 — APPROVED** [review]

File: server/meeting.go [hunk 1]
```diff
@@ -12,12 +12,14 @@ import (
...
 
-func (c *Client) CreateMeeting(creator *UserInfo, attendeesIDs []*UserInfo) (*msgraph.OnlineMeeting, error) {
+func (c *Client) CreateMeeting(creator *UserInfo, attendeesIDs []*UserInfo, subject string) (*msgraph.OnlineMeeting, error) {
 	ctx := context.Background()
 	start := time.Now()
 	end := start.Add(1 * time.Hour)
-	subject := "Mattermost Meeting"
 	attendees := []msgraph.MeetingParticipantInfo{}
+	if subject == "" {
+		subject = "MS Teams Meeting"
```
Comment (wiggin77) [inline server/meeting.go:21]:
```suggestion
		subject = "Microsoft Teams Meeting"
```

**Review by wiggin77 — APPROVED** [review]

**Comment by AayushChaudhary0001** [comment]
While testing this PR I have found 2 issues.

1. Multiple arguments allowed with spacing
2. No limit on length of the meeting topic

I am attaching screenshots supporting above statements, please take a look and see if we can fix them. Option 1 would be an enhancement(as MM considers each word as an argument), but option 2 should be fixed as it would be considered as a bug.

**For point 1:-**
image

**For point 2:-** 
image

**Comment by Kshitij-Katiyar** [comment]
> While testing this PR I have found 2 issues.
> 
> 1. Multiple arguments allowed with spacing
> 2. No limit on length of the meeting topic
> 
> I am attaching screenshots supporting above statements, please take a look and see if we can fix them. Option 1 would be an enhancement(as MM considers each word as an argument), but option 2 should be fixed as it would be considered as a bug.
> 
> **For point 1:-** image
> 
> **For point 2:-** image

@AayushChaudhary0001  I looked into your comment, and here is what I found:

- We want to support spaces in the meeting topic, so we are allowing multiple arguments. Another approach could be to take a single argument after the command, which can be the meeting topic enclosed in double quotes.
- I tested the length limit of the topic on the MS Teams meeting side and found that there is no restriction on the meeting topic length from their side, so we are also not imposing any limitation on the topic's length.

**Pull request merged**